ReactantPrototyping/ViewController.swift

```diff
@@ -21,25 +21,78 @@ final class ViewController: ControllerBase<Void, ExampleRootView> {
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         
-        navigationController?.hidesBarsOnTap = true
+//        navigationController?.hidesBarsOnTap = true
     }
 }
 
 
 final class ExampleRootView: ViewBase<Void, Void> {
 
-    private let labelInsideSafeArea = UILabel(text: "Hello Reactant!")
+//    private let labelInsideSafeArea = UILabel(text: "Hello Reactant!")
+    private let tableView = SimpleCollectionView<SimpleCell>(reloadable: false)
+    
+    override func afterInit() {
+        tableView.action
+            .subscribe(onNext: {
+                print($0)
+            }).disposed(by: lifetimeDisposeBag)
+    }
+    
+    override func update() {
+        tableView.componentState = .items(["Test1", "Test2", "Test3"])
+    }
 
     override func loadView() {
         children(
-            labelInsideSafeArea
+//            labelInsideSafeArea
+            tableView
         )
+        
+        tableView.collectionView.allowsMultipleSelection = true
+        tableView.collectionView.allowsSelection = true
+        tableView.collectionViewLayout.estimatedItemSize = CGSize(1)
+    }
+
+    override func setupConstraints() {
+//        labelInsideSafeArea.snp.makeConstraints { make in
+//            make.left.equalTo(fallback_safeAreaLayoutGuide).offset(20)
+//            make.top.equalTo(fallback_safeAreaLayoutGuide).offset(20)
+//        }
```

Comment:
Same question as line 24.

Also, shouldn't the constraints be separated by an empty line?